src/AuthModule/Components/Register/Register.tsx

```diff
@@ -1,11 +1,287 @@
+
+                  
+                  placeholder="Enter your phone number"
+                  aria-label="readonly input example"
+                />
+              </div>
+              {errors?.phoneNumber && (
+                <p className="  text-danger">{errors?.phoneNumber?.message}</p>
+              )}
+            </div>
+            <div className=" one-input-group       col-md-6 pe-lg-5 pe-md-3   my-2 col-12">
+              <div className="form-outline position-relative text-start d-flex flex-wrap">
+                <label htmlFor="password" className="w-100 form-label fw-medium mb-0">
+                  Password
+                </label>
+                <input
+                  {...register("password", passwordValidation)}
+                  type={!showPassword ? "text" : "password"}
+                  className="form-input form-control bg-transparent rounded-bottom-0  border-0 border border-bottom text-white p-1 pb-0 flex-grow-1"
+                    id="password"
+                  
+
+                  
+                  placeholder="Enter your password"
+                  aria-label="readonly input example"
+                />
+                <button className="input-group-text rounded-bottom-0  border-0  bg-transparent position-absolute mt-4 end-0 pt-2">
+                  <i
+                    className={`far ${
+                      showPassword ? "fa-eye-slash" : "fa-eye"
+                    } eye`}
+                    onClick={togglePasswordVisibility}
+                  ></i>
+                </button>
+              </div>
+              {errors?.password && (
+                <p className="  text-danger">{errors?.password?.message}</p>
+              )}
+            </div>
+            <div className=" one-input-group       col-md-6 ps-lg-5 ps-md-3   my-2 col-12">
+              <div className="form-outline position-relative text-start d-flex flex-wrap">
+                <label htmlFor="confirmPassword" className="w-100 form-label fw-medium mb-0">
+                  Confirm Password
+                </label>
+                <input
+                id="confirmPassword"
+                  {...register("confirmPassword", {
+                    required: "Confirm Password is required",
+                    validate: (value: string) =>
+                      value === confirmPassword || "Passwords do not match",
+                  })}
+                  type={!showConfirmPassword ? "text" : "password"}
+                  className="form-input form-control bg-transparent rounded-bottom-0   border-0 border border-bottom text-white p-1 pb-0 flex-grow-1"
+                    
+                  
+                                    
+                  
+                  placeholder="Confirm New Password"
+                  aria-label="readonly input example"
+                />
+                <button className="input-group-text rounded-bottom-0  border-0  bg-transparent position-absolute mt-4 end-0 pt-2">
```

Comment:
We need to add `type="button"` here, because currently when clicking on this button, it tries to submit the form.